packages/protocol/contracts/common/FeeCurrencyWhitelist.sol

```diff
@@ -25,6 +33,28 @@ contract FeeCurrencyWhitelist is IFeeCurrencyWhitelist, Ownable, Initializable {
     _transferOwnership(msg.sender);
   }
 
+  /**
+   * @notice Returns the storage, major, minor, and patch version of the contract.
+   * @return Storage version of the contract.
+   * @return Major version of the contract.
+   * @return Minor version of the contract.
+   * @return Patch version of the contract.
+   */
+  function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
+    return (1, 0, 1, 0);
+  }
+
+  // TODO fee tokens can't be removed
+
+  /**
+   * @dev Add a token to the whitelist
+   * @param tokenAddress The address of the token to add.
+   */
+  function addNonMentoToken(address tokenAddress) external onlyOwner {
```

Comment:
Should we also have removeNonMentoToken ?

Reply:
done